test/helpers.js

```diff
@@ -52,6 +53,8 @@ const mockTime = () => {
 
   // eslint-disable-next-line
   Date.prototype.getHours = () => HOUR;
+  // eslint-disable-next-line
+  Date.prototype.getMinutes = () => MIN;
```

Comment:
`getMinutes` also should be mocked. otherwise tests will won't pass on 50% of timezones. (e.g: +00:30 ,+01:30. +02:30 ...)

Reply:
Add please corresponding unmock code to unmockTime